setup/install/index.md

```diff
@@ -10,20 +10,23 @@ This section describes how to install and set up Spinnaker so that it can be con
 use in production. If you just want to evaluate Spinnaker without much work, one of the options
 in [Quickstart](/setup/quickstart/) might be a better choice.
 
+## What you'll need
+
+* A machine on which to install Halyard
+
+  This can be a local machine or VM (Ubuntu 14.04/16.04, Debian, or macOS), or
+  it can be a Docker container.
+
+* A Kubernetes cluster on which to install Spinnaker itself
```

Comment:
Is the intention here that we'll be strongly pushing users to install to Kubernetes going forward?

Reply:
Yes

Reply:
I don't know about "strongly pushing", however given a lack of particular preference from the user we want to highlight a primary path. This is strictly responsive to our installation option fan-out looking very large and overwhelming.